spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/data/neo4j/Neo4jProperties.java

```diff
@@ -72,6 +73,16 @@
 	 */
 	private Boolean openInView;
 
+	/**
+	 * Enables the direct usage of Neo4j native types in domain classes without applying
+	 * any converter. This includes all native date, time and duration of Neo4j 3.4+
+	 * (those are mapped to {@code java.time} types as well as the Neo4j spatial datatypes
```

Comment:
Configuration key description cannot use Javadoc tags ,see [the documentation](https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#configuration-metadata-annotation-processor).

Also, this documentation is quite long and unusual (compare this text with any other property in the codebase right now). Please consider rephrasing it to be more succinct.